### PR TITLE
Adds new integration [pantherale0/ha-familysafety]

### PR DESCRIPTION
**Pull request by pantherale0**
## Checklist

- [X] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [X] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [X] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [X] The actions are passing without any disabled checks in my repository.
- [X] I've added a link to the action run on my repository below in the links section.
- [X] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: <https://github.com/pantherale0/ha-familysafety/releases/tag/1.1.0>
Link to successful HACS action (without the `ignore` key): <https://github.com/pantherale0/ha-familysafety/actions/runs/7827730038>
Link to successful hassfest action (if integration): <https://github.com/pantherale0/ha-familysafety/actions/runs/7827730038>

### CODE REVIEW

**Comment by tjorim** [comment]
Please read the checklist before checking them, number 4 is not true: https://github.com/pantherale0/ha-familysafety/blob/main/.github/workflows/validate.yml#L37

**Comment by pantherale0** [comment]
forgotten about that key, apologies.

**Comment by tjorim** [comment]
No worries, looking good now.

File: integration [hunk 1]
```diff
@@ -740,6 +740,7 @@
   "p0l0/hapetwalk",
   "pail23/stiebel_eltron_isg_component",
   "pantherale0/ha-nintendoparentalcontrols",
+  "pantherale0/ha-familysafety",
```
Comment (ludeeus) [inline integration:743]:
```suggestion
  "pantherale0/ha-familysafety",
  "pantherale0/ha-nintendoparentalcontrols",
```

**Pull request merged**